modules/nw-multus-ipvlan-object.adoc

```diff
@@ -44,6 +46,10 @@ The following object describes the configuration parameters for the IPVLAN CNI p
 |`integer`
 |Optional: Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
 
+|`linkInContainer`
+|`boolean`
+|Optional: Specifies if the master interface is in the container network namespace or the main network namespace. To request the use of a container namespace master interface set the value to `true`.
```

Comment:
I'm a little confused by this parameter description. Is it set to true if the master interface is in one of those namespaces, and false if it's in neither? If that's the case, I'd suggest the following rewording:

```suggestion
|Optional: Specifies whether the master interface is in either the container network namespace or the main network namespace. To request the use of a container namespace master interface, set the value to `true`.
```

Or, is it set to true if the master interface is in the container network namespace and set to false if it's in the main network namespace? If this is the case, then I'd suggest the following:

```suggestion
|Optional: Specifies whether the master interface is in the container network namespace or in the main network namespace. To request the use of a container namespace master interface, set the value to `true`.
```

In both cases, I also replaced "if" with "whether", seeing as the ISG guidance (see "if versus whether" in the ["I" section of word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i)) suggests the use of "whether" in the case of discussing alternatives, and here it seems like you do need to choose between one of two options.

Reply:
revised